jobs/config.json

```diff
@@ -10349,7 +10349,7 @@
       "--gke-create-command=container clusters create --quiet --enable-kubernetes-alpha",
       "--gke-environment=test",
       "--provider=gke",
-      "--test_args=--ginkgo.focus=\\[Feature:(Audit|BlockVolume|PodPreset|ExpandPersistentVolumes)\\]|Networking --ginkgo.skip=Networking-Performance|IPv6 --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Feature:(Audit|BlockVolume|PodPreset|ExpandPersistentVolumes|VolumeSubpathExpansion)\\]|Networking --ginkgo.skip=Networking-Performance|IPv6 --minStartupPods=8",
```

Comment:
@BenTheElder @krzyzacy what does ci/k8s-stable1 point to?

Reply:
from https://github.com/kubernetes/test-infra#release-branch-jobs--image-validation-jobs, stable1 is latest 1.10 at the moment

Reply:
Thanks, so these two configs should not be modified